index.js

```diff
@@ -1,6 +1,11 @@
 var _ = require('lodash'),
   shell = require('gulp-shell'),
-  gutil = require('gulp-util');
+  gutil = require('gulp-util'),
+  util = require('util');
+
+require('dotbin');
```

Comment:
dotbin adds node_modules/.bin to the path during execution so that nodemon is found even when not installed globally.